grapple/types/images.py

```diff
@@ -22,7 +27,25 @@ class BaseImageObjectType(graphene.ObjectType):
     url = graphene.String(required=True)
     aspect_ratio = graphene.Float(required=True)
     sizes = graphene.String(required=True)
+
     collection = graphene.Field(lambda: CollectionObjectType, required=True)
+    placeholder_blur = graphene.String()
+
+    def resolve_placeholder_blur(self, info, **kwargs):
+        """
+        Get a low quality image placeholder of the original image.
+        This can directly be use in frontend frameworks like next.js
+        """
+        rendition = self.get_rendition("width-64|jpegquality-50")
+        with get_willow_image(rendition) as willow:
+            i = willow.grapple_blur()
+            tf = tempfile.SpooledTemporaryFile()
+            i.save_as_png(tf)
+            tf.seek(0)
+            hash = base64.b64encode(tf.read())
+            tf.close()
+
+        return "data:image/jpeg;base64,%s" % hash.decode("utf-8")
```

Comment:
on L43 we have `i.save_as_png(tf)`, but this says is it `image/jpeg` 🤔

Reply:
@fabienheureux I'd like to get this in before making a new release. any chance you could have a look at this?

and possibly add a comment to `SourceImageIOError`, `get_willow_image` and `open_file` that we should remove then when we drop support for Wagtail < 2.15

Reply:
actually, having thought more of it - this will create a temp file and blur it every single time which means this is potentially expensive on image heavy site 🤔

One way around it would be to register a new image filter so that the whole thing is saved as a rendition so we only do the base64 encoding

Reply:
Hey @zerolab, I put this PR back into draft mode, there are a few areas of improvements, especially finding the best rendition's size, and as you suggested, create a new image filter. 

I use this PR on production on a not-so-image-heavy project, and I noticed CPU usage peaks.
I did not have time to investigate yet but it might be related.

Reply:
Quick update on this PR, apart from the performance issues, it can break some features if the website uses GIF images. 
I am keeping this PR on hold before I find the time to test it properly, as it has many side effect I might have not think of.